File: kitsune/test/tapir/serial/cl-tto.f90
! REQUIRES: kitfc
!
! Check that the tapir target options specific to the serial tapir target are
! set correctly depending on the corresponding frontend options.
!
! Currently, there are no options specific to the serial tapir target. We just
! check that the tapir target ID is set correctly.
!
! RUN: %kitfc --tapir=serial -O2 -S -emit-llvm -o /dev/null %s \
! RUN:     -mllvm -print-tt-options \
! RUN:     | FileCheck %s -check-prefixes ALL
!
! ALL: Tapir target options
! ALL: Primary: serial

File: kitsune/test/tapir/qthreads/cl-tto.cpp
// Check that the options specific to the qthreads tapir target make it to the
// tapir target options.
//
// NOTE: Currently, there are no such options, so this is mostly just a
// placeholder and is around for consistency with the tests for the other tapir
// targets.
//
// RUN: %kitxx --tapir=qthreads -O2 -S -emit-llvm -o /dev/null %s \
// RUN:     -mllvm -print-tt-options \
// RUN:     | FileCheck %s -check-prefixes ALL
//
// ALL:          Tapir target options
// ALL:          Primary: qthreads
